src/WarcraftLegacies.Source/Quests/Scourge/QuestScouringofLordaeron.cs

```diff
@@ -11,18 +11,18 @@ public sealed class QuestScouringofLordaeron : QuestData
   {
     private readonly LegendaryHero _uther;
 
-    public QuestScouringofLordaeron(IEnumerable<Capital> capitalTargets, LegendaryHero uther) : base("Scouring of Lordaeron",
+    public QuestScouringofLordaeron(Capital capitalPalace, Capital stratholme, LegendaryHero uther) : base("Scouring of Lordaeron",
       "The Kingdom of Lordaeron must be eliminated to pave the way for the Legion's arrival.",
       @"ReplaceableTextures\CommandButtons\BTNGhoul.blp")
     {
       _uther = uther;
-      foreach (var capital in capitalTargets)
-        AddObjective(new ObjectiveCapitalDead(capital));
+      AddObjective(new ObjectiveControlCapital(capitalPalace, false));
+      AddObjective(new ObjectiveCapitalDead(stratholme));
     }
 
     /// <inheritdoc/>
     protected override string RewardFlavour =>
-      "The capital of Lordaeron has fallen, weakening Lordaeron's Champion.";
+      "The capitals of Lordaeron has fallen, weakening Lordaeron's Champion.";
```

Comment:
Lorewise I don't think Stratholme is a capital. Maybe like, the "major cities of Lordaeron have fallen"?
Also it's an old problem but some explanation of why this weakens Uther would be a huge improvement.

Reply:
Major cities will sound better!

Reply:
Could remove the weakning part to be honest. and just remove the quest, and make him unrevivable when the 2 bases are done